HACKING.md

```diff
@@ -116,6 +116,13 @@ Run all tests through the test execution wrapper with
 $ ./test.py --mode={debug,release}
 ```
 
+or, if you are using `dbuild`, you need to build the tests and then you can run them at will:
+
+```console
+$ ./tools/toolchain/dbuild ninja {debug,release,dev}-test
```

Comment:
`ninja {mode}-build` actually seems to be enough (`{mode}-test` runs all tests in addition)

```suggestion
$ ./tools/toolchain/dbuild ninja {debug,release,dev}-build
```

I just tried, removing the whole build dir and using `ninja dev-build`, and I can then run `./test.py --mode dev` without issues.

Also, this is in fact not specific to just dbuild, this should behave the same way even without using the dbuild, so I'm wondering if we really need a specific section for the dbuild or if we should update the non-dbuild version instead (and mention it is the same when using the dbuild).

One thing to mention explicitly would perhaps be that if using the dbuild for building, the dbuild also needs be used for running the `./test.py` (and vice versa).